collections/forms/src/validators/__tests__/integer.test.js

```diff
@@ -36,6 +37,8 @@ describe('validator: integer', () => {
             0.23456,
             5.987,
             1e-12,
+            '4.0',
+            '4,0',
```

Comment:
Does the regex also catch multiple trailing zeros? We should make sure values like `'1.000000'` are also rejected, and add a test for that.

Reply:
yes it does .. added a test